File: API.Foodie/API.Foodie/Model/Order.cs
namespace API.Foodie.Model;

public class Order
{
    public int Id { get; set; }
    public DateTime OrderDate { get; set; }
    public DateTime DeliveryDate { get; set; }
    public decimal TotalPrice { get; set; }
    public string Status { get; set; }
    public string Address { get; set; }

    public int AppUserId { get; set; }
    public AppUser AppUser { get; set; }
    public List<OrderDish> Dishes { get; set; }
}

File: webClient.Foodie/src/app/models/order.ts
import { OrderDish } from "./order-dish";

export interface Order {
    id: number;
    orderDate: Date;
    deliveryDate: Date;
    totalPrice: number;
    status: string;
    address: string;

    email: string;
    firstName: string;
    lastName: string;
    phoneNumber: string;

    dishes: OrderDish[];
}
